packages/dapp-connector/src/ScriptInitializers/types.ts

```diff
@@ -0,0 +1,6 @@
+import { OpaqueString } from '@cardano-sdk/util';
+
+const typedString = (value: string) => value;
+
+export type ApiName = OpaqueString<'ApiName'>;
+export const ApiName = (target: string) => typedString(target);
```

Comment:
Is this returning an ApiName? :thinking: 
```suggestion
export const ApiName = (target: string) => target as ApiName;
```